pkg/projects/Microsoft.NETCore.Console/project.json

```diff
@@ -0,0 +1,51 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore": "5.0.1-rc3-24126-00",
+    "Microsoft.NETCore.ConsoleHost": "1.0.0-rc3-24126-00",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.0.2-rc3-24126-00",
```

Comment:
Honestly I don't know who all is using it right now so for now I'm blinding bringing to get us out of TFS but we can consider removing later.